scripts/rook.sh

```
@@ -123,7 +131,9 @@ function teardown_rook() {
 	kubectl delete -f "${ROOK_URL}/toolbox.yaml"
 	kubectl delete -f "${ROOK_URL}/cluster-test.yaml"
 	kubectl delete -f "${ROOK_URL}/operator.yaml"
+	kubectl delete -f "${ROOK_URL}/crds.yaml"
```

this was already applied above?

---

I didnt get @nixpanic , its in teardown() , other one is in deploy?

---

this might fail if the rook < 1.5

---

@Madhu-1 I was thinking on the same line, however my understanding was that , if we failed to delete we ignore the error. Isnt it not the case?

---

am not sure, but needs to be tested. for safer side we can delete with `--ignore-not-found` flag.

---

added ignore flag for safer side